kv/kvserver/protectedts/ptstorage: TestStorage failed

kv/kvserver/protectedts/ptstorage.TestStorage [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15722362?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15722362?buildTab=artifacts#/) on release-24.1 @ [dd69d0e8ad0b93c70f7355dcdc65c59d7526558c](https://github.com/cockroachdb/cockroach/commits/dd69d0e8ad0b93c70f7355dcdc65c59d7526558c):

Fatal error:

```
panic: test timed out after 4m57s
running tests:
	TestStorage (4m57s)
	TestStorage/Protect_-_unlimited_bytes (14s)

```
Stack: 

```
goroutine 139699 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestStorage/Protect_-_no_targets_withDeprecatedSpans
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/6e3b57159e391a80fe5e9a5259d87206/logTestStorage_Protect_-_no_targets_withDeprecatedSpans3449157039
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    storage_test.go:528: -- test log scope end --
=== RUN   TestStorage/Protect_-_too_many_spans_withDeprecatedSpans_withMetaTable/4
=== RUN   TestStorage/Protect_-_unlimited_bytes_withMetaTable
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/6e3b57159e391a80fe5e9a5259d87206/logTestStorage_Protect_-_unlimited_bytes_withMetaTable910553365
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    storage_test.go:528: -- test log scope end --
=== RUN   TestStorage/MarkVerified_withMetaTable/1
=== RUN   TestStorage/Protect_-_too_many_bytes_withDeprecatedSpans_withMetaTable/3
=== RUN   TestStorage/Protect_-_too_many_bytes_withMetaTable/0

pkg/kv/kvserver/protectedts/ptstorage_test/pkg/kv/kvserver/protectedts/ptstorage/storage_test.go:414: (protectOp.run)
	NOTICE: .Clock() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Clock() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
=== RUN   TestStorage/Protect_-_too_many_spans_withDeprecatedSpans_withMetaTable/8
=== RUN   TestStorage/Protect_-_too_many_spans_withDeprecatedSpans_withMetaTable/9
=== RUN   TestStorage/Protect_-_too_many_spans_withMetaTable/2
=== RUN   TestStorage/MarkVerified_withMetaTable
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/6e3b57159e391a80fe5e9a5259d87206/logTestStorage_MarkVerified_withMetaTable2238141217
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    storage_test.go:528: -- test log scope end --
=== RUN   TestStorage/Protect_-_already_verified_withDeprecatedSpans_withMetaTable/0

pkg/kv/kvserver/protectedts/ptstorage_test/pkg/kv/kvserver/protectedts/ptstorage/storage_test.go:123: (init.func2)
	NOTICE: .Clock() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Clock() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
=== RUN   TestStorage/Protect_-_unlimited_bytes/3
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStorage.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-39642

--- a/pkg/kv/kvserver/protectedts/ptstorage/BUILD.bazel
+++ b/pkg/kv/kvserver/protectedts/ptstorage/BUILD.bazel
@@ -30,7 +30,7 @@ go_library(
 
 go_test(
     name = "ptstorage_test",
-    size = "medium",
+    size = "large",
     srcs = [
         "main_test.go",
         "storage_test.go",

